### ISSUE TITLE
Submit budgie-desktop into Fedora proper?

### ISSUE THREAD

**Issue by Conan-Kudo** [alunux/rpm-specfiles#2]
Is there a reason you haven't submitted Budgie into Fedora itself? The last package review for budgie stalled out a while back, so a new package review would be welcomed!

**Comment by alunux** [comment]
I'm sorry because I didn't respond this issue because the git repo of budgie-desktop has low activities. Is it still acceptable to submit Budgie into Fedora?

**Comment by Conan-Kudo** [comment]
Absolutely. It's still maintained by @ikeydoherty, so there's no reason not to.

**Comment by TheElectronWill** [comment]
+1 on this

**Comment by ikeydoherty** [comment]
It has low activity because Budgie 10.x is basically LTS and will only see brief periods of activity, but mostly just minor fixes (no real new features)

**Comment by ForlornWongraven** [comment]
This would be really helpful.

**Comment by alunux** [comment]
With the current development of Budgie, I think I will stay with copr.

**Comment by Conan-Kudo** [comment]
What about the current development of Budgie means that you will stay in copr only?

**Comment by alunux** [comment]
@Conan-Kudo Yes. Sorry about this, but the Mutter issue has given me some worries because I am not familiar with Vala (not interested). I see pantheon desktop also suffered because of that issue[1], finally there are two packages of mutter in repository.

Also, it's already a year after the latest release (Budgie 10.4) and it fails to build on Fedora 29+.

[1] https://decathorpe.com/2018/09/04/call-for-help-pantheon-on-fedora-29.html

**Comment by Conan-Kudo** [comment]
There are already pull requests for porting to GNOME 3.30 platform: https://github.com/solus-project/budgie-desktop/pulls

**Comment by alunux** [comment]
@Conan-Kudo But, you can't apply those patches to Budgie 10.4 tarball, it fails. So many changes happened. Any advice?

**Comment by Conan-Kudo** [comment]
I could probably help create backport patches for this. Let me make some time for that.

**Comment by alunux** [comment]
Thank you @Conan-Kudo. I will give my best too.
